src/go/rdctl/cmd/factoryReset.go

```diff
@@ -17,19 +17,27 @@ limitations under the License.
 package cmd
 
 import (
-	"bytes"
 	"encoding/json"
+	"errors"
 	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"syscall"
 
+	dockerconfig "github.com/docker/docker/cli/config"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/factoryreset"
 	"github.com/spf13/cobra"
 )
 
-var factoryResetData struct {
-	KeepSystemImages bool `json:"keepSystemImages"`
-}
-
 var removeKubernetesCache bool
 
+// Note that this command supports a `--remove-kubernetes-cache` flag,
+// but the server takes an optional flag meaning the opposite (as per issues 1701 and 2408)
```

Comment:
Please refer to URLs rather than issue numbers; we've been through too many projects that change issue tracking systems…

Reply:
ok